PDF.java

```diff
@@ -0,0 +1,133 @@
+//You will need apache pdfbox, apache fontbox, apache commonsloggings, jdbc JARs
+//So far this will just generate a pdf with no values, still need to pull info from DB
+
+import java.sql.*;
+import java.io.IOException;
+import org.apache.commons.logging.*;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.PDPage;
+import org.apache.pdfbox.pdmodel.PDPageContentStream;
+import org.apache.pdfbox.pdmodel.font.PDFont;
+import org.apache.pdfbox.pdmodel.font.PDType1Font;
+
+public class PDF {
+	private PDDocument document = new PDDocument();
+	private PDFont font = PDType1Font.TIMES_ROMAN;
+	private PDPage page = new PDPage();
+	private ResultSet rs;
+	private Connection con;
+	private final String conURL; //URL for connecting to DB
+	private String conUSER;//Username for connecting to DB
+	private String conPW;//Password for connecting to DB
+	private String filePath;//Path to save the pdf
+	
+	public PDF(String conURL, String conUSER, String conPW, String filePath) {
```

Comment:
I think we'll standardize the DB object which contains a connection, statement, and callable statement as members and access to the database. So the file path will be the only parameter.

Reply:
Are we even putting it in the DB? I thought we decided we would just save it on the local machine. We can even make the email function delete the file afterwards.